build.gradle.kts

```diff
@@ -9,6 +9,7 @@ buildscript {
 
   dependencies {
     classpath("com.android.tools.build:gradle:${versions.agp}")
+    // TODO: When bumping Kotlin version to 1.5.10. Remove the kgp-leak-patcher plugin
```

Comment:
This is the wrong place to add the comment because someone bumping up the version will not touch this section unless the version bump also requires a change to the maven coordinates. The right place to put this is at the version declaration site.

Does this make sense?